src/renderer/services/face-recognition/share-assets-with-opencv.js

```diff
@@ -0,0 +1,42 @@
+import { createFolder } from '@/services/misc'
```

Comment:
These functions are used to copy some projects assets required by opencv on the machine. As the project assets are packed with `asar` with electron build, they are not available from other programs like opencv, [seen here](https://simulatedgreg.gitbooks.io/electron-vue/content/en/using-static-assets.html) in the last section.

Reply:
Here the app.getPath function would help too 👍